script-test/utils/timeutilstest.js

```diff
@@ -25,6 +25,35 @@ require(
           testDurationToSeconds(duration);
         }
       });
+
+      describe('Calculate Sliding Window Seek Offset', function () {
+        beforeEach(function () {
+          jasmine.clock().mockDate(new Date('2019-10-22T10:55:24.952Z'));
+        });
+
+        afterEach(function () {
+          jasmine.clock().uninstall();
+        });
+
+        it('should return the relative time in seconds including the time a user spent seeking', function () {
+          var time = 4895.142446990982;
+          var dvrInfoRangeStart = new Date('2019-10-22T08:54:24.006Z') / 1000;
```

Comment:
Is it possible to use some more reader friendly dates here? It's difficult to follow all the variables and see why the expectation is the expectation.